inenv/inenv.py

```diff
@@ -171,6 +179,18 @@ def setup_venv(venv_name, verbose, quiet):
             subprocess_call(['pip', 'install', dep], verbose)
 
 
+def jump_to_req_dir(venv_name):
+    """Jumps to the directory of the reqs file"""
+    deps = activate_and_get_deps(venv_name)
+    for dep in deps:
+        if dep.startswith('file:'):
+            dep = rel_path_to_abs(dep.split('file:')[1])
```

Comment:
lets actually add a directory directive to the .ini file since file:{} is not required as part of deps.